samples/drivers/watchdog/src/main.c

```diff
@@ -88,12 +93,19 @@ void main(void)
 	wdt_config.window.min = 0U;
 	wdt_config.window.max = WDT_MAX_WINDOW;
 
+#if (WDT_DISABLE_CALLBACK - 0)
```

Comment:
What's with the `- 0`?

Reply:
Avoids a syntax error when `WDT_DISABLE_CALLBACK` is undefined.

Reply:
The condition then expands to:

    #if - 0

which is false.